pcloudcv/utility/parseArguments.py

```diff
@@ -31,37 +31,30 @@ def getParams(self):
     def readConfigFile(self, file):
         data_file = open(file, 'r').read()
         complete_data = json.loads(data_file)
-        self.data = complete_data['config']
-        self.exec_name = complete_data['exec']
-        self.maxim = int(complete_data['maxim'])
+        self.data = complete_data.get('config')
+        self.maxim = int(complete_data.get('maxim'))
+        self.server_port = int(complete_data.get('server_port'))
```

Comment:
Why is 'exec' parameters? If you notice in config.json.example - there is a 'exec' parameter.